Assets/Countly/Plugins/CountlySDK/Helpers/SafeIDGenerator.cs

```diff
@@ -16,5 +12,18 @@ public string GenerateValue()
             return CountlyUtils.SafeRandomVal();
         }
     }
-}
-
+
+    /// <summary>
+    /// CustomIdProvider class is for testing purposes
+    /// </summary>
+    public class CustomIdProvider : ISafeIDGenerator
```

Comment:
Its name is CustomIdProvider but it provides viewId? name and logic irrelevant

Reply:
May not be the best naming tbh you are correct. First intention was creating something that can do multiple things but ended up with a custom view id provider. Name can be updated to CustomViewIdProvider or sth